apps/cowswap-frontend/src/modules/hooksStore/pure/AddCustomHookForm/CustomDappLoader/index.tsx

```diff
@@ -15,6 +15,26 @@ interface ExternalDappLoaderProps {
   setManifestError: Dispatch<SetStateAction<string | React.ReactNode | null>>
 }
 
+const TIMEOUT = 5000
+
+const fetchWithTimeout = async (url: string, options: any) => {
+  const controller = new AbortController()
+  const timeoutId = setTimeout(() => {
+    controller.abort()
+  }, options.timeout)
+
+  try {
+    const response = await fetch(url, { signal: controller.signal, ...options })
```

Comment:
Please, use  existing `getTimeoutAbortController()` from `@cowprotocol/common-utils`

Reply:
Thanks for reviewing. Addressed this now.